infrastructure_files/base.setup.env

```diff
@@ -39,8 +35,6 @@ export NETBIRD_AUTH0_AUDIENCE
 export NETBIRD_LETSENCRYPT_EMAIL
 export NETBIRD_MGMT_API_PORT
 export NETBIRD_MGMT_API_ENDPOINT
-export NETBIRD_MGMT_GRPC_API_PORT
-export NETBIRD_MGMT_GRPC_API_ENDPOINT
```

Comment:
we need to update the configure.sh script and remove all references of the removed variables